integration-cli/docker_cli_rename_test.go

```diff
@@ -39,8 +45,7 @@ func (s *DockerSuite) TestRenameRunningContainer(c *check.C) {
 }
 
 func (s *DockerSuite) TestRenameRunningContainerAndReuse(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	out, _ := dockerCmd(c, "run", "--name", "first_name", "-d", "busybox", "top")
+	out, _ := dockerCmd(c, "run", "--name", "first_name", "-d", "busybox", sleepCmd, "60")
```

Comment:
Curious, what's the reason for switching to "sleep"? Or isn't `top` supported on Windows? IIRC, `sleep` was replaced by `top` for a reason; see https://github.com/docker/docker/issues/12188, https://github.com/docker/docker/pull/12208, and https://github.com/docker/docker/pull/13912